sklearn/datasets/tests/test_20news.py

```diff
@@ -94,3 +95,20 @@ def test_20news_vectorized():
     assert bunch.data.shape == (11314 + 7532, 130107)
     assert bunch.target.shape[0] == 11314 + 7532
     assert bunch.data.dtype == np.float64
+
+
+def test_20news_normalization():
+    try:
+        X = datasets.fetch_20newsgroups_vectorized(normalize=False,
+                                                   download_if_missing=False)
+        X_ = datasets.fetch_20newsgroups_vectorized(normalize=True,
+                                                    download_if_missing=False)
+    except IOError:
+        raise SkipTest("Download 20 newsgroups to run this test")
+
+    # to dense because np.allclose fails otherwise.
```

Comment:
`from sklearn.utils.testing import all_close_dense_sparse` will allow you to do that without densifying.